### ISSUE TITLE
NuGet client 2.12 on Ubuntu Linux

### ISSUE THREAD

**Issue by define-private-public** [NuGet/Home#3382]
I was trying to install the `Microsoft.AspNetCore.Http` package from NuGet.  When I tried to, I got this message:

```
The 'System.Text.Encodings.Web 4.0.0' package requires NuGet client version '2.12' or above, but the current NuGet version is '2.8.7.0'.
```

So how do I update my NuGet client to 2.12 on Ubuntu Linux?  running `apt-get update/upgrade` didn't do it.

**Comment by rrelyea** [comment]
Are you trying to run NuGet.exe on mono on Ubuntu Linux?
Or something else?

**Comment by define-private-public** [comment]
Yeah, it was through the MonoDevelop IDE.  I got that from the Mono project's PPA.

**Comment by rrelyea** [comment]
This sounds like a question for MonoDevelop IDE, I think. Will try to get their attention...via an email with a pointer to this item.

**Comment by rrelyea** [comment]
I've been told that filing a bug in Bugzilla and assigning to Jo Shields is the best way to work with the owners of this. As such, I'm going to close this issue as external. Please file the bug in Bugzilla, and then put a link to it here.

**Comment by mrward** [comment]
[Bugzilla bug](https://bugzilla.xamarin.com/show_bug.cgi?id=44043)

NuGet 2.12 is included with Xamarin Studio 6.0.2. Currently there are no Linux packages available for MonoDevelop 6 and you will need to compile the code from [GitHub](https://github.com/mono/monodevelop/).

If you are using MonoDevelop 5.10 and do not want to compile MonoDevelop 6 from source you can try replacing the NuGet binaries in the /usr/lib/monodevelop/AddIns/MonoDevelop.PackageManagement directory. The NuGet 2.12 binaries are available from [GitHub](https://github.com/mono/nuget-binary/tree/2.12). The Microsoft.Web.XmlTransform.dll and NuGet.Core.dll files are the ones used by MonoDevelop.
